node/src/BaseClient.ts

```diff
@@ -7672,4 +7679,46 @@ export class BaseClient {
             throw err;
         }
     }
+
+    /**
+     * Update the current connection password with a new password.
+     *
+     * This method is useful in scenarios where the server password has changed or when utilizing short-lived passwords for enhanced security.
+     * It allows the client to update its password to reconnect upon disconnection without the need to recreate the client instance.
+     * This ensures that the internal reconnection mechanism can handle reconnection seamlessly, preventing the loss of in-flight commands.
+     *
+     * This method updates the client's internal password configuration and does not perform password rotation on the server side.
+     *
+     * @param password - The new password to update the current password, or `null` to remove the current password.
+     * @param reAuth - If `true`, the client will re-authenticate immediately with the new password. If `false`, the new password will be used for the next connection attempt.
+     * @returns {Promise<GlideString>} A promise that resolves to the response from the server.
+     *
+     * @example
+     * ```typescript
+     * await client.updateConnectionPassword("newPassword", true) // "OK"
+     * ```
+     */
+    async updateConnectionPassword(password: string | null, reAuth: boolean) {
```

Comment:
This should be in stadalone client only, because cluster client has no configurable retry

Reply:
Same for other clients too